lib/tasks/verify-origami-json.js

```diff
@@ -26,8 +26,8 @@ function origamiJson(config) {
 						if (origamiJson.origamiType !== 'imageset' && origamiJson.origamiType !== 'module' && origamiJson.origamiType !== 'service') {
 							result.push('The origamiType property needs to be set to either "imageset", "module" or "service"');
 						}
-						if (origamiJson.origamiVersion !== 2) {
-							result.push('The origamiVersion property needs to be set to 2, this version of Origami Build tools only supports v2 of the Origami component specification.');
+						if (typeof origamiJson.origamiVersion !== 'string' || origamiJson.origamiVersion.split('.')[0] !== '2') {
+							result.push('The origamiVersion property needs to be set to "2.0" or higher, this version of Origami Build tools only supports v2 of the Origami component specification.');
```

Comment:
non-blocking suggestion: might be helpful to mention that it needs to be a string here? Or maybe we should add a more specific error for when 2.0 is set as a Number?

Reply:
Added a message that says it must be a string. It will print both errors if they provide a number, so they will see that it must be the string and the format the string must be in.